Adjusted the desktop file for Debian.

This patch by Kęstutis Biliūnas.
This patch will not be submitted upstream.

--- a/Packaging/fontforge.desktop
+++ b/Packaging/fontforge.desktop
@@ -8,7 +8,7 @@
 Comment[fr]=Un constructeur des polices
 Comment[zh_TW]=FontForge 描邊字型編輯器
 Exec=fontforge
-Icon=fontforge.png
+Icon=ffanvil32.xpm
 Terminal=false
 Type=Application
 Categories=Application;Graphics;
